cloudinit/tests/test_util.py

```diff
@@ -189,6 +189,30 @@ def test_mount_is_ro(self, m_mount_info):
         self.assertEqual(is_rw, False)
 
 
+class TestUptime(CiTestCase):
+
+    @skipUnlessFreeBSD()
```

Comment:
I agree with Ryan that this test isn't appropriate here; it's not a unit test.

I'm definitely open to introducing some more integration-level testing for FreeBSD, but I think having it interleaved with unit tests will lead to confusion.

Reply:
aaaaah, now i understand.